system/jlib/jmetrics.hpp

```diff
@@ -176,6 +176,11 @@ class jlib_decl CounterMetric : public MetricVal
     {
         value.fastAdd(val);
     }
+
+    void clear()
```

Comment:
The current usage is to reset the stats between jobs by a thor that stays running, which is almost analogous to a system restart. But I can see the argument either way. It thor at the same time clears the cache (which I think it may) then resetting the stats does kind-of make sense,